rust/src/main.rs

```diff
@@ -15,6 +15,11 @@ use std::io;
 
 extern crate xi_core_lib;
 extern crate xi_rpc;
+extern crate fern;
```

Comment:
again, by convention (and also by alphabetical ordering) I'd prefer the non-xi crates above the internal crates